src/presenters/teaser-presenter.js

```diff
@@ -90,7 +90,7 @@ const TeaserPresenter = class TeaserPresenter {
 		//use package title as display tag if article belongs to package
 		let packageArticle = this.data.containedIn
 		if (packageArticle && packageArticle[0] && packageArticle[0].title) {
-			return Object.assign(this, { prefLabel: packageArticle[0].title });
+			return Object.assign(this, { prefLabel: packageArticle[0].title }, { relativeUrl: packageArticle[0].relativeUrl});
```

Comment:
couldn't you just do this in one object: `return Object.assign(this, { prefLabel: packageArticle[0].title,  relativeUrl: packageArticle[0].relativeUrl});`